hashintel/hash
# Add `<Skeleton/>` to `@hashintel/design-system` & use it for address block

## 🌟 What is the purpose of this PR?

Implementing skeleton loading state for blocks which requires it.


## 🔗 Related links

- [Asana task](https://app.asana.com/0/1204000740778938/1204086167143851/f) _(internal)_

## 🔍 What does this change?

- Creates `Skeleton` component at `@hashintel/design-system`
- Implements loading state for `address` block


## 🐾 Next Steps

- [x] Check other external API blocks to see if they require skeleton loading state

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Run address block, visually confirm it has loading state as shown below


## 📹 Demo


https://user-images.githubusercontent.com/39553853/227010501-868738ef-7baf-4229-9bd5-872f5b7e1346.mov
